static/src/javascripts/projects/common/modules/commercial/hosted-gallery.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'bean',
+    'bonzo',
+    'fastdom',
+    'common/utils/$'
+], function (
+    bean,
+    bonzo,
+    fastdom,
+    $
+) {
+
+
+    function init() {
+        var $scrollEl = $('.hosted-gallery__scroll-container');
+        var $images = $('.hosted-gallery__image');
+        var noImages = $images.length;
```

Comment:
noImages is a really confusing name - given that we only use it once, we could just use it there and make it explicit? or `numberOfImages`?